base/pkg/query.jl

```diff
@@ -359,7 +359,7 @@ function prune_versions(reqs::Requires, deps::Dict{String,Dict{VersionNumber,Ava
             vmaskp[vn] = falses(luds)
         end
         for (vn,a) in fdepsp
-            vmind = findfirst(uniqdepssets, a.requires)
+            vmind = findfirst(Base.equalto(a.requires), uniqdepssets)
```

Comment:
Why  `Base.`?

Reply:
Shouldn't be necessary since it's exported?